dmalloc_argv.c

```diff
@@ -2412,8 +2398,7 @@ static	void	do_list(argv_t *grid, const int arg_c, char **argv,
       }
       
       /* check for special usage value */
-      if (strncmp(USAGE_ARG, *arg_p + LONG_PREFIX_LENGTH, len) == 0
-	  || strncmp(HELP_ARG, *arg_p + LONG_PREFIX_LENGTH, len) == 0) {
```

Comment:
Why was the HELP_ARG support removed?

Reply:
Because it block line 2452/2437 I suppose it should print argv_help_string in this case.

Reply:
Yeah can we revert this?  They are printing different information.  This is printing the usage message while below brings the help line.  I want --usage and --help to display the usage strong.

Reply:
So, you prefer to remove?
https://github.com/j256/dmalloc/pull/69/files#diff-324a2b5ba7c2f719c853831ed17df55f8ce68bf781d2c4ae98734f29946b575bR2437-R2453
and argv_help_string(no other place reference it) since it's impossible to enter that branch.